Src/AutoFoq/AutoFoqCustomization.fs

```diff
@@ -18,7 +18,8 @@ type AutoFoqCustomization(relay: ISpecimenBuilder) =
         AutoFoqCustomization(
             FilteringSpecimenBuilder(
                 MethodInvoker(
-                    FoqMethodQuery()),
+                    FoqMethodQuery(
+                        Fixture())),

```

Comment:
An `ICustomization` is normally something you apply to a given `IFixture` instance. Wouldn't this mean that the default `AutoFoqCustomization` applied to an `IFixture` would contain *another* `Fixture` instance? Isn't that going to be confusing to users?